### DISCUSSION TITLE
PasteIntoFile 讓你可以直接在資料夾貼上圖片

### DISCUSSION THREAD

**Discussion by sys511613** [1101-GS4719-C/Python-Community-GS4719-C#612] [優秀專案]
# 專案網址：[PasteIntoFile](https://github.com/EslaMx7/PasteIntoFile)
上次有人介紹了win + shift + s 螢幕截圖的方法
經過螢幕截圖以後，擷取下來的圖片會被存在剪貼簿中。
如果要把他存成圖片的話就要點進去剪取工具並另存新檔，或是用小畫家等圖片編輯軟體打開後另存到資料夾，這樣下來也需要花費不少時間。

### **PasteIntoFile**這個東西就能讓你快速的把圖片或文字(或各種檔案格式)直接從剪貼簿另存成檔案
![image](https://user-images.githubusercontent.com/73024991/168971125-4ab93920-ae6c-44d4-92e0-5e5f3c2ba2ae.png)
☝🏼這些都是他支援的格式

專案的[README.md](https://github.com/EslaMx7/PasteIntoFile/blob/master/README.md)有寫他的安裝方法，非常簡單！
裝好以後，只要複製了什麼東西，到你想要放的資料夾中按右鍵，選Paste Into File
![2022-05-18 143757](https://user-images.githubusercontent.com/73024991/168974026-189bc942-6acb-40f3-a57b-18a1d8a4ace3.png)

輸入好檔名和檔案格式後，按下Save，他就會神奇的出現在資料夾中囉
![image](https://user-images.githubusercontent.com/73024991/168976500-d7dc3ac2-6694-44fb-a2a9-6116577690cd.png)
![image](https://user-images.githubusercontent.com/73024991/168976779-dd35a562-0037-459d-a195-885c1a512aad.png)
☝🏼像這樣

**Comment by gnsJhenJie** [comment]
存在剪貼簿的截圖真的很尷尬，常常不知道怎麼把他弄出來，有這個程式感覺會方便很多。

**Comment by awkward-willy** [comment]
這工具真的讓截圖方便許多
先前我都是用 win + shift + s 後點擊跳出來的視窗再選取儲存
現在即使沒點到也可以不用到小畫家調整了

**Comment by 11timyyy** [comment]
電腦雖然本身有截圖功能，但剛剛試了一下感覺方便很多!
